### ISSUE TITLE
MultiMC launches pack with bad environment variables

### ISSUE THREAD

**Issue by Conner-Anderson** [simibubi/Above-and-Beyond#134]
MultiMC, which is a a launcher recommended to install the modpack with, will not load worlds by default.
Symptoms:
- Unusually long launch time, hanging on a full loading bar on the beige Mojang screen.
- Disconnects after multiple minutes of "loading terrain" when connecting to a server. Does not produce a server error.
- Generates local, singleplayer world extremely slowly then freezes at some low percentage (~5%). One time, it failed to get past the "generating terrain" title

Used Java 1.8.0

The issues were solved by copying the default Java parameters from ATLauncher to MultiMC. These are:

`-XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M`

Also, I set maximum RAM to 4096 and PermGen to 256.

This was quite un-user-friendly, and I only figured it out because I've played (and modded) Minecraft since 2011.

**Comment by simibubi** [comment]
Thanks for reporting- Is there anything the pack itself can do to fix this?

**Comment by Conner-Anderson** [comment]
I poked around, and I've found no way a pack can specify JVM variables on launch. Most launchers allow the user to customize them on a per-instance or global basis, as I did. Most launchers have default variables that work with Above and Beyond: MultiMC does not.

Solutions: remove MultiMC from the launcher list on the wiki, or include instructions for MultiMC. If you remove the launcher, those who already use it should realize that it is possible and are probably familiar with how to fix the issue already. If you include instructions, here is a suggestion of its contents:
"
If you use the MulltiMC Launcher, go to Settings menu then the Java tab and paste this line into the JVM Arguments box:
`-XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M`

Also set Maximum Memory Allocation to 4096 MiB.

"
Thanks for your work on this splendid modpack! My friends and are are having a grand ol' time playing it.
